js/exercise-04.js

```diff
@@ -0,0 +1,19 @@
+/*
+    What would each of these lines of code do:
+
+    1. true && console.log("moo");
+    2. false && console.log("moo moo?")
+    3. true || console.log("hello friend");
+    4. false || console.log("bye friend");
+
+    Once you see the results, write in a comment why it works that way
+*/
+
+true && console.log("moo");
```

Comment:
This exercise is not properly explained. You have to show every case output and explain the reason of its behaviour.

```suggestion
true && console.log("moo"); // output and explanation
```

FIX: show output and justify it